encoding/gocode/gen.go

```diff
@@ -123,10 +123,10 @@ func GenerateTypesOpenAPI(sch thema.Schema, cfg *TypeConfigOpenAPI) ([]byte, err
 	}
 
 	return PostprocessGoFile(GenGoFile{
-		Path:     fmt.Sprintf("%s_type_gen.go", sch.Lineage().Name()),
-		Appliers: applyFuncs,
-		In:       []byte(gostr),
-		ErrIfAdd: !cfg.IgnoreDiscoveredImports,
+		Path:                    fmt.Sprintf("%s_type_gen.go", sch.Lineage().Name()),
+		Appliers:                applyFuncs,
+		In:                      []byte(gostr),
+		IgnoreDiscoveredImports: !cfg.IgnoreDiscoveredImports,
```

Comment:
Doesn't it look weird that `IgnoreDiscoveredImports: !IgnoreDiscoveredImports`? 🤔 

Also, based on the underlying behavior, I'm not sure if the name is clear enough. I guess nothing that cannot be clarified with some extra explanation in form of docs, if needed.

Reply:
That's a copy-pasta mistake, good catch!